asciinema/commands/record.py

```diff
@@ -43,7 +44,12 @@ def execute(self):
                 self.print_error("can't write to %s" % self.filename)
                 return 1
 
-            if os.stat(self.filename).st_size > 0 and not append:
+            if os.stat(self.filename).st_size > 0 and self.overwrite:
+                self.print_info("removing the file at %s" % self.filename)
```

Comment:
It would probably be just fine to not print anything here, as the overwrite has been explicitly requested. Also, the overwrite could be implemented as truncate operation on file, without actually removing it, and the fact we actually do remove the file completely is implementation detail ;)

What do you think?

Reply:
How can i truncate the file?
Should i use https://linux.die.net/man/1/truncate and add a new method in the __pty_recioder.py__ file?
From my current point of view, truncating a file is much more complicated.

Reply:
I wasn't clear, sorry. I didn't mean we should truncate it, removing s fine. I meant we could remove this `print_info` line :)